airbyte-integrations/connectors/source-hibob/main.py

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+
+import sys
+
+from airbyte_cdk.entrypoint import launch
+from source_hibob import SourceHibob
+
+if __name__ == "__main__":

```

Comment:
It's best to just import `run` and run `run` instead of duplicating it. Please refactor this as well.